### PR TITLE
fix: use right DTD logo in the footer

### LINKED ISSUE #162: DTD logo in the footer
We should replace the MID logo in the footer with the DTD one, like we have at developers.italia.it

### PR DESCRIPTION
**Pull request by bfabio**
Fix #162.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -136,10 +136,7 @@ <h2 class="bd-logo-subtitle text-white">Registra in Developers Italia il tuo str
 
         <div class="d-flex pt-2">
           <div class="px-2 team-logo--icon">
-            <img src="assets/logo-mid-white.svg" alt="Dipartimento per la Trasformazione Digitale" />
-          </div>
-          <div class="px-2 border-white border-left team-logo--text text-white">
```
Comment (libremente) [inline public/index.html:141]: @bfabio should we also remove https://github.com/italia/developers-italia-onboarding/blob/23079447ceb02e56b3324aa892c5799ec271a0e9/public/assets/css/main.css#L55?

Comment (bfabio) [reply public/index.html:141]: @libremente nice catch

**Comment by libremente** [comment]
Why there's no vercel here?

**Comment by bfabio** [comment]
@libremente this is not a static site, there's a server running

**Comment by libremente** [comment]
> @libremente this is not a static site, there's a server running

oh yeah, we need staging then! :)

**Review by libremente — APPROVED** [review]
LGTM

**Review by sebbalex — APPROVED** [review]

**Pull request merged**